channels-rke2.yaml

```diff
@@ -4,8 +4,10 @@ channels:
 appDefaults:
   - appName: rancher
     defaults:
-      - appVersion: '> 2.6.0-0 < 2.6.99-0'
+      - appVersion: '> 2.6.0-0 < 2.6.5-0'
```

Comment:
I feel like one of these should be an inclusive range. I know we never ship with a `-0` (and `< 2.6.5-0` is effectively synonymous with `<= 2.6.4` since the `-0` prerelease identifier in `2.6.5-0` means "the earliest possible thing that could be considered part of the `2.6.5` release line), but according to strict semver, this means that `2.6.5-0` is excluded (i.e. has no default), and 2.6.5-rc1 is included for 1.23.x. While we will never ship a `-0` version (IIRC the earliest thing used is always an `-alpha1`, we also do some weird things in rancher internally regarding prerelease versions. If 1.23.x should be the default for `v2.6.5-rc1` and up, I think it should be `appVersion: '>= 2.6.5-0 < 2.6.99-0'` instead so it's explicit which boundary it should be including.

Reply:
Good points! PR is updated, please review it again.